test/e2e/storage/persistent_volumes.go

```diff
@@ -323,7 +323,12 @@ var _ = utils.SIGDescribe("PersistentVolumes", func() {
 				e2estatefulset.DeleteAllStatefulSets(c, ns)
 			})
 
-			ginkgo.It("should be reschedulable [Slow]", func() {
+			// testsuites/multivolume tests can now run with windows nodes
+			// This test is not compatible with windows because the default StorageClass
+			// doesn't have the ntfs parameter, we can't change the status of the cluster
+			// to add a StorageClass that's compatible with windows which is also the
+			// default StorageClass
+			ginkgo.It("should be reschedulable [Slow] [LinuxOnly]", func() {
```

Comment:
Should all tests that use default storageclass be skipped, not just this one?

Reply:
sure, I moved the `[LinuxOnly]` annotation to `gingko.Describe`